### DISCUSSION TITLE
[privacy] unwanted DNS/network connections to google

### DISCUSSION THREAD

**Discussion by ghjkl99** [Alex313031/thorium-legacy#104] [Q&A]
@Alex313031 Thank you so much for making this available for Win7 too. i do have chrome://settings/syncSetup options to disabled, safebrowsing disabled and incognito mode, but am seeing DNS lookups for: update.googleapis.com, safebrowsing.googleapis.com, chromewebstore.googleapis.com, edgedl.me.gvt1.com, clients2.googleusercontent.com without actually visiting those sites. Please could you clarify ? Would like to not have these network connections too. Thanks.

**Comment by gz83** [comment]
As long as Google services are retained, it is impossible to completely avoid links to Google servers. Additionally, we currently have no plans to ungoogle.

Perhaps you could also try using a firewall to block both outbound and inbound connections between Thorium and Google's servers?

**Reply by ghjkl99** [reply]
Thanks @gz83. Sorry, no offense meant, but can thorium be based on ungoogled-chromium (or use all those privacy-respecting patches) so thorium can be even better than ungoogled-chromium with such features that ungoogled-chromium misses ?. Thanks.

**Reply by gz83** [reply]
Thanks for the suggestion, but we currently have no plans to do this (ungoogled).
